x-pack/platform/packages/shared/kbn-ai-assistant/src/conversation/conversation_view.tsx

```diff
@@ -176,6 +182,7 @@ export const ConversationView: React.FC<ConversationViewProps> = ({
             showLinkToConversationsApp={false}
             onConversationUpdate={handleConversationUpdate}
             navigateToConversation={navigateToConversation}
+            onConversationDuplicated={onConversationDuplicated}
```

Comment:
```suggestion
            onConversationDuplicate={handleConversationDuplicate}
```
To be consistent with `handleConversationUpdate`